scripts/system_field_text_update.rb

```diff
@@ -0,0 +1,2 @@
+rsf = IO.read(ARGV[0])
+File.write(ARGV[0], rsf.gsub!(/\t(user)\t((\w|-)+)/m, "\tsystem\tfield:" + '\2'))
```

Comment:
If I'm not mistaken, you can remove the parenthesis wrapping `user` and avoid the third group capture by using `[]`:

```
File.write(ARGV[0], rsf.gsub!(/\tuser\t([\w-]+)/m, "\tsystem\tfield:" + '\1'))
```